docs/source/notebooks/conditional-autoregressive-model.ipynb

```diff
@@ -3110,7 +3110,7 @@
     "        n, m = W.shape\n",
     "        self.n = n\n",
     "        self.median = self.mode = self.mean = 0\n",
-    "        super(Sparse_CAR, self).__init__(*args, **kwargs)\n",
+    "        super().__init__(*args, **kwargs)\n",
```

Comment:
Does this still work without the arguments passed in the old versions?

Reply:
I don't know if it works with versions of Python older than 3.6, although 3.6 is the oldest version currently supported by PyMC3 and this notebook lists Python 3.8 as the version it was run on so my thinking was that it was OK

Reply:
Ow ok, didn't know that not passing the args to `super` worked now!